### ISSUE TITLE
New command `Create New Terminal in Editor Area (With Profile)`

### ISSUE THREAD

**Issue by CombeeMike** [microsoft/vscode#208306]
When opening a new terminal during dev, I usually want this to open in the editor and therefore use the command `Terminal: Create New Terminal in Editor Area`.

This is mostly fine but I sometimes have to open a terminal with a profile != my default one. This is only possible with `Terminal: Create New Terminal (With Profile)` which doesn't open the terminal in my editor but in the "view" (a panel at the bottom in my case).

Whilst this is okish, I'd usually prefer to also open this in the editor.

I realize that setting `terminal.integrated.defaultLocation` to `editor` would solve this but this also changes all tasks to run in the editor which is a no-go for me and there's no way (that I know of) to force my tasks to keep running in the "view" with this setting.

# In short

An additional `Create New Terminal in Editor Area (With Profile)` to the existing `Terminal: Create New Terminal`, `Terminal: Create New Terminal (With Profile)` & `Terminal: Create New Terminal in Editor Area` would really make sense in general and would be really helpful to my personal workflows.

**Comment by Tyriar** [comment]
I'm adding support for a `location` arg in https://github.com/microsoft/vscode/pull/208436, so you can add a keybinding for your desired case. For commands I think there are already enough and if anything we should reduce the amount and make them more general purpose.

**Comment by CombeeMike** [comment]
@Tyriar Perfect, I was actually thinking about suggesting this as an alternative but forgot about it when writing the ticket.

Thanks!

**Comment by CombeeMike** [comment]
@Tyriar Since you're actively working on this:

Is there any kind of public documentation about available arguments for existing commands? I did not find anything when searching for it last week & co pilot also wasn't of much help.

**Comment by Tyriar** [comment]
@CombeeMike glad to hear this works for you 🎉 

Commands don't generally have documentation outside of maybe one off examples on the website or the intellisense provided in the editor:

![image](https://github.com/microsoft/vscode/assets/2193314/f2ac59dc-00ff-4bd5-9b39-823bf82db9d9)

I don't think the keybindings editor supports dealing with args unfortunately.

I've created https://github.com/microsoft/vscode-docs/issues/7146 to add a little example to the [terminal profiles](https://code.visualstudio.com/docs/terminal/profiles) page 👍
